src/Ocelot/DownstreamRouteFinder/Finder/DownstreamRouteFinder.cs

```diff
@@ -18,7 +19,7 @@ public DownstreamRouteFinder(IUrlPathToUrlTemplateMatcher urlMatcher, IPlacehold
             _placeholderNameAndValueFinder = urlPathPlaceholderNameAndValueFinder;
         }
 
-        public Response<DownstreamRoute> Get(string upstreamUrlPath, string upstreamQueryString, string httpMethod, IInternalConfiguration configuration, string upstreamHost)
+        public async Task<Response<DownstreamRoute>> GetAsync(string upstreamUrlPath, string upstreamQueryString, string httpMethod, IInternalConfiguration configuration, string upstreamHost)
```

Comment:
i usually prefer adding "Async" suffix after any async method name because it add clarity to me and also because of name changes, compiler complains about method not found and i get to know the places where i need to add await and prevent the returned value from Task<T> to T, especially if var is used.